app/routes.js

```diff
@@ -76,6 +76,16 @@ export const routes = (authProvider) => {
     res.render('pages/compare/template.njk', addConfig({ context, user: req.user }));
   });
 
+  router.get('/compare/document', async (req, res) => {
+    logger.info('downloading olution comparison document');
+    const response = await getDocument({
+      endpointLocator: 'getDocument',
+      options: { documentName: 'comparesolutions.xlsx' },
```

Comment:
can we rename this to `compare-solutions` we have the covid19 doc hyhenated

Reply:
ok but this is what Sam suggested on the json contract

Reply:
ok I'll put a comment on the json contract task.

Reply:
Should be fine to rename it to `compare-solutions`